parser_test.go

```diff
@@ -5370,11 +5373,10 @@ func TestCustomFunctionResolveReadQuery(t *testing.T) {
 	t.Parallel()
 
 	type testCase struct {
-		name                string
```

Comment:
no need to have two levels of `mustFail` now. All tests will fail at the resolving level